src/lib/common/limits.h

```diff
@@ -75,4 +75,86 @@
 #define ALARM_MGR_NOTIFICATION_ERROR_LOG_SAMPLING     1
 #define ALARM_MGR_BAD_INPUT_LOG_SAMPLING              1
 
+/* ****************************************************************************
+*
+* Message definitions - 
+*/
+#define MAX_STA_MSG_SIZE (20 * 1024)           // 20 KB (HTTP request static buffer)
+#define MAX_DYN_MSG_SIZE (8 * 1024 * 1024)     // 8 MB  (maximum length of the HTTP request dynamic buffer)
+
+/* ****************************************************************************
+*
+* PAYLOAD_MAX_SIZE - 
+*/
+#define PAYLOAD_MAX_SIZE   (1 * 1024 * 1024)
```

Comment:
1 MB? A comment should explain (similar to what is being done with for instance MAX_STA_MSG_SIZE)